examples/nodejs-server/README.md

```diff
@@ -1,3 +1,30 @@
-## Node.js Example Project
+## Node.js Example Project with Encryption
 
-See the [guide](https://app-config.netlify.app/guide/node/example.html).
+### Installation
+Start by installing dependancies
+```sh
+yarn
+```
+Next if you haven't done so already, initialize your local keychain 
+```sh
+yarn app-config secret init
+```
+In order to trust yourself to this project, you can export your public certificate by running
+```
+yarn app-config secrets export my_public_key.asc
+```
+In order to trust yourself to this project run [more on trusting users](https://app-config.netlify.app/guide/intro/encryption.html#trusting-users)
```

Comment:
vuepress will resolve links to markdown files, should be `../../guide/intro/encryption.md#trusting-users`